File: WebGeo/GeoRent.Domain/Entities/UserImage.cs
using System;
using System.Runtime.Serialization;

namespace GeoRent.Domain.Entities
{
    [DataContract]
    public class UserImage
    {
        public UserImage()
        {
            idUserImage = Guid.NewGuid();
        }

        [DataMember]
        public Guid idUserImage { get; set; }
        [DataMember]
        public String path { get; set; }
        [DataMember]
        public int resource { get; set; }
        [DataMember]
        public int order { get; set; }
        [DataMember]
        public virtual User idUser { get; set; }
    }
}

File: app/UserImage.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class UserImage extends Model
{
    protected $hidden = ['idUser','remember_token','created_at','updated_at'];
    
    protected $primaryKey = "idUserImage";

    protected $table = 'userImage';

    protected $fillable = ['path','resource','order'];

    public $timestamps = false;

    public function user()
    {
        return $this->belongsTo('App\User','idUser');
    } 
}
